[GUI] [Trivial] Rebranding work

JSKitty opened the pull request:
This PR rebrands the vast majority of the ZENZO Core v2.0 codebase, user-viewable elements such as the QT and CLI wallets, as well as the codebase itself, methods, comments and variables for consistency.

This PR should NOT change the functionality of ZENZO Core, but simply rebrand and rename the codebase.

---
This should be fairly easy to review, even for non-devs, as there isn't "code" added/removed, it's simply "PIVX --> ZENZO", "PIV --> ZNZ", etc. If anything looks inconsistent, out-of-place, or something was missed; Please leave a review!

akshaynexus commented:
Hashing algo is Xevan.not quark,as shown in block.cpp

if(nVersion < 4)
        return XEVAN(BEGIN(nVersion), END(nNonce));

Commit:
[Core] Rebrand thread names

This commit renames PIVX and Bitcoin threads to ZENZO

Commit:
[GUI] [Trivial] Rebrand name & ticker strings

This commit renames PIVX strings (And ONLY strings) to ZENZO and ZNZ, this mostly affects the QT and CLI wallets and rebrands strings that the user sees in their applicable frontends. (Either GUI or Console)

Commit:
[GUI] [Trivial] Rebrand comments + minor strings

This commit renames PIVX and PIV strings to ZENZO and ZNZ, as well as renaming some strings that were missed in the previous commit.

Commit:
[Trivial] Rebrand codebase: methods, variables, etc

This commit renames PIVX and PIV variables, methods, enums and more to ZENZO and ZNZ, this should not change the functionality of the code, but is a simple rebrand of the codebase itself for consistency.

YuurinBee reviewed (changes requested):
Commented with suggestions, awaiting @JSKitty 's feedback/approval before final approval + merge.

JSKitty commented:
PR compiles well and the majority of the needed codebase is rebranded, any PIVX-branded Zerocoin code will be removed entirely in the near future, and minor nits will be left for an additional PR.
